namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/dps-smoke-test.tf

```diff
@@ -0,0 +1,69 @@
+
+module "dps_smoketest_queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=4.4"
+
+  environment-name          = var.environment-name
+  team_name                 = var.team_name
+  infrastructure-support    = var.infrastructure-support
+  application               = var.application
+  sqs_name                  = "dps_smoketest_dev_hmpps_queue"
+  encrypt_sqs_kms           = "true"
+  message_retention_seconds = 600
+  namespace                 = var.namespace
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "aws_sqs_queue_policy" "dps_smoketest_queue_policy" {
+  queue_url = module.dps_smoketest_queue.sqs_id
+
+  policy = <<EOF
+  {
+    "Version": "2012-10-17",
+    "Id": "${module.dps_smoketest_queue.sqs_arn}/SQSDefaultPolicy",
+    "Statement":
+      [
+        {
+          "Effect": "Allow",
+          "Principal": {"AWS": "*"},
+          "Resource": "${module.dps_smoketest_queue.sqs_arn}",
+          "Action": "SQS:SendMessage",
+          "Condition":
+                      {
+                        "ArnEquals":
+                          {
+                            "aws:SourceArn": "${module.hmpps-domain-events.topic_arn}"
+                          }
+                        }
+        }
+      ]
+  }
+
+EOF
+
+}
+
+resource "kubernetes_secret" "dps_smoketest_queue" {
+  metadata {
+    name      = "dps-smoketest-queue-secret"
+    namespace = "dps-toolkit"
```

Comment:
and put back in all the secrets that have temporarily been commented out / removed since these namespaces have been migrated to live